build.sbt

```diff
@@ -12,21 +11,26 @@ lazy val jvmSdkVersion = "1.1.0"
  */
 
 lazy val `commercetools-sunrise-payment` = (project in file("."))
-  .aggregate(`common`, `payone`)
+  .aggregate(`common`, `payone-adapter`)
   .settings(javaUnidocSettings ++ commonSettings : _*)
+  .settings(
+      libraryDependencies ++= Seq (
+        "com.novocode" % "junit-interface" % "0.11",
+        "org.assertj" % "assertj-core" % "3.4.1"
+      )
+  )
```

Comment:
If you don't want to have IT here (which makes sense), you can simply put `configCommonTestSettings("test")` to keep the other test settings as well as the dependencies.